packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -220,10 +225,29 @@ const View = BaseView.extend({
       if (ChildView === CommunicationPreferencesView) {
         return areCommunicationPrefsVisible;
       }
+      if (ChildView === SubscriptionView) {
+        return this._isSubscriptionsManagementVisible();
+      }
       return true;
     });
   },
 
+  /**
+   * Should the subscriptions management panel be displayed?
+   *
+   * @returns {Boolean}
+   * @private
+   */
+  _isSubscriptionsManagementVisible () {
+    if (! this._subscriptionsManagementEnabled) {
+      return false;
+    }
+    if (!this._subscriptionsManagementLanguages.includes(this._language)) {
```

Comment:
nit: can you add a space after the `!`?

Reply:
This was resolved.

Reply:
Heh, yeah, I noticed a lint failure here pretty much 2 seconds before this comment 😅